### PR TITLE
feat: add robot-simulator exercise

### PR DESCRIPTION
**Pull request by vaeng**

### CODE REVIEW

**Review by vaeng — COMMENTED** [review]
I'm not sure if this is really a 5. Took me a while to get the syntax right, but in the end it is not that difficult.

File: exercises/practice/robot-simulator/robot-simulator.ua [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Get a number's prime factors
+# RobotPosition | x y cardinal
+Create ←  ⊙(⍤ "Please implement Create" 0 0 "")
+
```
Comment (ErikSchierboom) [inline exercises/practice/robot-simulator/robot-simulator.ua:4]:
```suggestion
```

File: exercises/practice/robot-simulator/robot-simulator.ua [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Get a number's prime factors
+# RobotPosition | x y cardinal
```
Comment (ErikSchierboom) [inline exercises/practice/robot-simulator/robot-simulator.ua:2]:
```suggestion
# RobotPosition | X Y Cardinal
```

**Review by ErikSchierboom — APPROVED** [review]
Nice. I think 5 is good

**Pull request merged**